agents_infra/agents_infra/utils/communication.py

```diff
@@ -0,0 +1,183 @@
+import logging
+
+from logtools import maybe_log_message
+
+from ..utils.network import (check_http_health, find_first_healthy_url,
+                             is_tcp_reachable, ping_url)
+from ..utils.timestamp import get_current_time
+
+revert_interval = 900
```

Comment:
Consider making `revert_interval` a parameter of the `AgentCommunication` class (possibly with a default), or a configuration value, so you can adjust it per `agent/controller`.